### DISCUSSION TITLE
Title field and Content field returning null in wpgraphql when quering custom post type

### DISCUSSION THREAD

**Discussion by Dimswati** [wp-graphql/wp-graphql#2863] [Q&A] [answered]
I have created a custom post type with CPT UI and registered support for title, editor and featuredImage and made it show in graphql, yet when I query for this title and content they return while featured Image -> sourceUrl returns a url

query GetPages {
  services {
    edges {
      node {
        content(format: RAW)
        databaseId
        date
        slug
        status
        title(format: RAW)
        uri
        featuredImage {
          node {
            sourceUrl(size: LARGE)
          }
        }
      }
    }
  }
}

results

{
  "data": {
    "services": {
      "edges": [
        {
          "node": {
            "content": null,
            "databaseId": 85,
            "date": "2023-07-20T07:40:30",
            "slug": "wooden-flooring",
            "status": "publish",
            "title": null,
            "uri": "/services/wooden-flooring/",
            "featuredImage": {
              "node": {
                "sourceUrl": "https://architect.paulmbatia.com/wp-content/uploads/2023/07/service-image-2.jpg"
              }
            }
          }
        },
        {
          "node": {
            "content": null,
            "databaseId": 83,
            "date": "2023-07-20T07:39:55",
            "slug": "texture-painting",
            "status": "publish",
            "title": null,
            "uri": "/services/texture-painting/",
            "featuredImage": {
              "node": {
                "sourceUrl": "https://architect.paulmbatia.com/wp-content/uploads/2023/07/service-image-1.jpg"
              }
            }
          }
        }
      ]
    }
  },
  "extensions": {
    "debug": [
      {
        "type": "DEBUG_LOGS_INACTIVE",
        "message": "GraphQL Debug logging is not active. To see debug logs, GRAPHQL_DEBUG must be enabled."
      }
    ]
  }
}

**Comment by justlevine** [comment, marked as answer]
Hey @Dimswati 

The `RAW` format is only available for authenticated requests. You can [toggle authentication in the GraphiQL IDE](https://www.wpgraphql.com/docs/wp-graphiql#toggle-between-public-and-authenticated-requests) to ensure it works, and drop the arg (or change it to `RENDERED`) and you should be able to get your data back.

**Comment by Dimswati** [comment]
thanks
